ddtrace/appsec/_asm_request_context.py

```diff
@@ -151,7 +151,7 @@ def finalise(self):
             callbacks = GLOBAL_CALLBACKS.get(_CONTEXT_CALL, []) if env.must_call_globals else []
             env.must_call_globals = False
             if env is not None and env.callbacks is not None and env.callbacks.get(_CONTEXT_CALL):
```

Comment:
`env.callbacks.get(_CONTEXT_CALL)` is being called twice. Perhaps the first call can be assigned and re-used?

Reply:
it's a get on a native dict, so very fast. I prefer to not introduce performance improvement in this PR, I opened another PR for that to be merged later: https://github.com/DataDog/dd-trace-py/pull/8617
This is only bug fix.